model/hardware_schema.json

```diff
@@ -27,7 +31,7 @@
       },
       "series": {
         "type": "string",
-        "description": "the series of this component, example: rtx4080"
+        "description": "the series of this component, example: gtx1080"
       },
       "share": {
         "type": "number",
```

Comment:
ici j'ajouterai : This share properties should be by defaut to 100% and if you do share the component indicate the percentage of share.